Python/ki/__init__.py

```diff
@@ -1178,7 +1178,13 @@ def OnScreenCaptureDone(self, image):
 
         PtDebugPrint(u"xKI.OnScreenCaptureDone(): Saving image to \"{}\".".format(tryName), level=kWarningLevel)
         if "saveAsPNG" in dir(image):
```

Comment:
Well sure, that's how I'd write it today.  😉 

A better question is whether we even need to maintain the old JPEG branch here anymore, as even Cyan's client supports PNGs.  The purpose of this was to silently upgrade to a PNG without breaking the script when run by older clients.  Anyone running a client that old now is probably also running a fork of the scripts, right?